src/components/Header/Header.tsx

```diff
@@ -44,7 +44,7 @@ export const Header: React.FC<IHeader> = ({
 
   const isXsDown = useMediaQuery(theme.breakpoints.down('xs'))
 
-  const routes = ['swap', 'pool']
+  const routes = ['swap', 'pool', 'IDO']
```

Comment:
IDO page is something not included in scope of recruitment task, only component and storybook demo of it